### PR TITLE
BC-5928 - Fix language keys for external tools

### PR DESCRIPTION
**Pull request by VikDavydiuk**
# Short Description

QA note:
Change tested  on env:
https://nbc-bc-5928-fix-language-keys.cd.dbildungscloud.dev/
No issues found. All texts for all languages have been corrected. 
More info in ticket: https://ticketsystem.dbildungscloud.de/browse/BC-5928
Update 06.12.2023
I tested again on env:
https://nbc-bc-5928-fix-language-keys.cd.dbildungscloud.dev/
No issues found. All texts for all languages have been corrected, all comments included.

## Links to Ticket and related Pull-Requests

- https://ticketsystem.dbildungscloud.de/browse/BC-5928

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Review by blazejpass — DISMISSED** [review]

File: src/locales/en.json [hunk 1]
```diff
@@ -173,8 +173,8 @@
...
-  "components.administration.externalToolsSection.info": "This area allows third-party tools to be seamlessly integrated into the cloud. With the functions provided, tools can be added, existing tools can be updated or tools that are no longer needed can be removed. By integrating external tools, the functionality and efficiency of the cloud can be extended and adapted to specific needs.",
-  "components.administration.externalToolsSection.description": "The school-specific parameters for the external tool are configured here. After saving the configuration, the tool will be available within the school.<br><br>\nDeleting a configuration will make the tool withdrawn from school.<br><br>\nFurther information is available in our <a href=\"https://docs.dbildungscloud.de/pages/viewpage.action?pageId=246055610\" target=\"_blank\">Help section on external tools</a>.",
+  "components.administration.externalToolsSection.info": "This area allows third-party tools to be seamlessly integrated into the cloud. With the functions provided, tools can be added, existing tools can be updated or tools that are no longer needed can be removed.",
```
Comment (hoeppner-dataport) [inline src/locales/en.json:176]: Isn't a translation for "Lehrkräfte müssen die Tools dann zu Kursen hinzufügen." missing here and in es.json and uk.json?

Comment (VikDavydiuk) [reply src/locales/en.json:176]: The ticket description does not mention this https://ticketsystem.dbildungscloud.de/browse/BC-5928

File: src/locales/en.json [hunk 2]
```diff
@@ -173,8 +173,8 @@
...
-  "components.administration.externalToolsSection.info": "This area allows third-party tools to be seamlessly integrated into the cloud. With the functions provided, tools can be added, existing tools can be updated or tools that are no longer needed can be removed. By integrating external tools, the functionality and efficiency of the cloud can be extended and adapted to specific needs.",
-  "components.administration.externalToolsSection.description": "The school-specific parameters for the external tool are configured here. After saving the configuration, the tool will be available within the school.<br><br>\nDeleting a configuration will make the tool withdrawn from school.<br><br>\nFurther information is available in our <a href=\"https://docs.dbildungscloud.de/pages/viewpage.action?pageId=246055610\" target=\"_blank\">Help section on external tools</a>.",
+  "components.administration.externalToolsSection.info": "This area allows third-party tools to be seamlessly integrated into the cloud. With the functions provided, tools can be added, existing tools can be updated or tools that are no longer needed can be removed. Teachers must then add the tools to courses.",
```
Comment (Loki-Afro) [inline src/locales/en.json:176]: 'must', not 'could'?

Comment (VikDavydiuk) [reply src/locales/en.json:176]: Please, read comments from Ingwert Paulsen in the ticket https://ticketsystem.dbildungscloud.de/browse/BC-5928

**Review by hoeppner-dataport — APPROVED** [review]

**Review by blazejpass — APPROVED** [review]

**Pull request merged**